library/src/main/java/com/novoda/downloadmanager/WaitForDownloadServiceThenPerform.java

```diff
@@ -0,0 +1,52 @@
+package com.novoda.downloadmanager;
+
+import android.support.annotation.Nullable;
+
+import com.novoda.notils.logger.simple.Log;
+
+final class WaitForDownloadServiceThenPerform {
```

Comment:
Should be able to use this in other places like the `LiteDownloadManagerDownloader` which should improve the readability and maintainability a little bit!